boskos/cmd/cli/cli.go

```diff
@@ -93,11 +100,12 @@ scripts with a simple interface.`,
 		Args: cobra.NoArgs,
 	}
 	root.PersistentFlags().StringVar(&options.serverURL, "server-url", "", "URL of the Boskos server")
+	root.PersistentFlags().StringVar(&options.username, "username", "", "Username used to access the Boskos server")
+	root.PersistentFlags().StringVar(&options.password, "password", "", "The path to password file used to access the Boskos server")
 	root.PersistentFlags().StringVar(&options.ownerName, "owner-name", "", "Name identifying the user of this client")
 	for _, flag := range []string{"server-url", "owner-name"} {
 		if err := root.MarkPersistentFlagRequired(flag); err != nil {
-			fmt.Println(err)
-			os.Exit(1)
+			logrus.WithError(err).Fatalf("Failed to mark flag '%s' required", flag)
```

Comment:
Users do not want to see `logrus` output. They want to see `Println`.

Reply:
AFAIK, logrus can be configured to use a console formatter

Reply:
(Not sure why we needed to change this at all, though)

Reply:
Reverted.
The reason that I used logrus ... censor :)